lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -192,24 +192,29 @@ function getNodeTextLength(node) {
 /**
  * @param {Driver} driver
  * @param {LH.Crdp.DOM.Node} node text node
- * @returns {Promise<NodeFontData['cssRule']|undefined>}
+ * @returns {Promise<?NodeFontData['cssRule']|undefined>}
  */
 async function fetchSourceRule(driver, node) {
-  const matchedRules = await driver.sendCommand('CSS.getMatchedStylesForNode', {
-    nodeId: node.nodeId,
-  });
-  const sourceRule = getEffectiveFontRule(matchedRules);
-  if (!sourceRule) return undefined;
-
-  return {
-    type: sourceRule.type,
-    range: sourceRule.range,
-    styleSheetId: sourceRule.styleSheetId,
-    parentRule: sourceRule.parentRule && {
-      origin: sourceRule.parentRule.origin,
-      selectors: sourceRule.parentRule.selectors,
-    },
-  };
+  try {
+    const matchedRules = await driver.sendCommand('CSS.getMatchedStylesForNode', {
+      nodeId: node.nodeId,
+    });
+    const sourceRule = getEffectiveFontRule(matchedRules);
+    if (!sourceRule) return undefined;
+
+    return {
+      type: sourceRule.type,
+      range: sourceRule.range,
+      styleSheetId: sourceRule.styleSheetId,
+      parentRule: sourceRule.parentRule && {
+        origin: sourceRule.parentRule.origin,
+        selectors: sourceRule.parentRule.selectors,
+      },
+    };
+  } catch (err) {
+    Sentry.captureException(err, {tags: {gatherer: 'FontSize'}});
```

Comment:
Do we want to sentry this?

Reply:
I'm not sure there's any value here. We know nodes can disappear it's not really helping us to understand the volume.